paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -104,21 +107,43 @@ class PaparazziPlugin : Plugin<Project> {
         val nonTransitiveRClassEnabled =
           (project.findProperty("android.nonTransitiveRClass") as? String).toBoolean()
 
+        // local resources
+        val localResourceFiles = project
+          .files(variant.sourceSets.flatMap { it.resDirectories })
+          .asFileTree
+
+        // library resources
+        // https://android.googlesource.com/platform/tools/base/+/96015063acd3455a76cdf1cc71b23b0828c0907f/build-system/gradle-core/src/main/java/com/android/build/gradle/tasks/MergeResources.kt#875
+        val libraryResourceFiles = variant.runtimeConfiguration
+          .incoming
+          .artifactView { config: ArtifactView.ViewConfiguration ->
+            config.attributes { container: AttributeContainer ->
+              container.attribute(
+                AndroidArtifacts.ARTIFACT_TYPE,
+                AndroidArtifacts.ArtifactType.ANDROID_RES.type
+              )
+            }
+          }
+          .artifacts
+          .artifactFiles
+          .asFileTree
```

Comment:
possible optimization for a later PR: maybe removing the FileTree optimization which will walk the directory contents when the FileCollection is dereferenced and instead, we'd walk the directory contents within the ResourceRepositories.  Let's leave as is for now, but also keep this in mind.

The difference would be in the contents of `resources.txt`.

without fileTree:
```src/main/res,../(module)/build/intermediates/packaged_res/debug```

with fileTree:
```src/main/res/drawable/foo.png,src/main/res/values/strings.xml, ../(module)/build/intermediates/packaged_res/debug/drawable/bar.png,../(module)/build/intermediates/packaged_res/debug/values/values.xml```